python/lsst/ip/isr/calibType.py

```diff
@@ -516,9 +558,9 @@ def fromTable(cls, tableList):
         metadata = table.meta
         inDict = dict()
         inDict['metadata'] = metadata
-        inDict['detectorName'] = metadata['DETECTOR']
-        inDict['detectorSerial'] = metadata['DETECTOR_SERIAL']
-        inDict['instrument'] = metadata['INSTRUME']
+        inDict['detectorName'] = metadata.get('DET_NAME', None)
```

Comment:
This is an attempt to match the usage in DMTN-148, as well as already existing calibs.  I would prefer this to be `DET_ID` to be explicit.  The reason this is shifted into the base class updateMetadata is because I was being inconsistent in my use of keywords.

Reply:
As long as it id is there as well then that's great 🙂